core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_Team.hpp

```diff
@@ -115,44 +115,68 @@ class ParallelFor<FunctorType, Kokkos::TeamPolicy<Properties...>,
     // mode but works in the Debug mode.
 
     // Maximum active teams possible.
-    int max_active_teams = OpenMPTargetExec::MAX_ACTIVE_THREADS / team_size;
-    // nteams should not exceed the maximum in-flight teams possible.
-    const auto nteams =
-        league_size < max_active_teams ? league_size : max_active_teams;
+    int max_active_teams = omp_get_max_teams();
```

Comment:
Why not `min(nteams, omp_get_max_team())`

Reply:
That `min` is happening in `resize_scratch` and the resulting value is set in `omp_set_num_teams` . 
The value is then being accessed here using `omp_get_max_teams` rather than setting another variable that is passed between routines.